script/travis/before_install.sh

```diff
@@ -59,6 +59,30 @@ then
             exit 1
         fi
     fi
+
+    if [ "${ALPAKA_CI_STDLIB}" == "libstdc++" ]
+    then
+        if [ "${CXX}" == "clang++" ]
+        then
+            if [ ! -z ${ALPAKA_CXX_STANDARD+x} ]
+            then
+                if (( "${ALPAKA_CXX_STANDARD}" >= 17 ))
```

Comment:
I just realized: even if `ALPAKA_CXX_STANDARD` is set to `11`, with a new compiler this might select C++14 or even C++17 or C++2a with `target_compile_features`